core/common/pom.xml

```diff
@@ -57,7 +57,8 @@
     </dependency>
     <dependency>
       <groupId>io.netty</groupId>
-      <artifactId>netty-all</artifactId>
+      <artifactId>netty</artifactId>
+      <version>3.7.0.Final</version>
```

Comment:
why change the version only for this specific module?

Reply:
This module uses `ConcurrentIdentityMap` implementation from netty which is dropped after this version. And that's the only use of netty from this module.

Actually we don't need an explicit netty dependency anymore. I can have another PR to remove netty dependencies throughout the code. (Locally I tested that removing netty dependencies, except this one, has no effect on build.)

Reply:
did you create a tech dept issue for this?

Reply:
If we can remove the direct dependency on Netty that would be helpful.

Reply:
https://github.com/Alluxio/alluxio/issues/9449